src/modules/organizations/components/edit-details.jsx

```diff
@@ -6,12 +6,18 @@ import LinksContainer from '../containers/links-container';
 import ImageSelector from '../../common/containers/image-selector';
 
 const EditDetails = (props) => {
+  let listedAtDate;
+
   if (!props.organization) {
     return (
       <div>Loading...</div>
     );
   }
 
+  if (props.organization.listed) {
```

Comment:
shouldn't matter if all is working as intended, but if what we'll be looking for to determine `listedAtDate` is the `listed_at` attribute I'm thinking we set the conditional check based on that attribute as well

Reply:
Sounds good - should we adjust the attribute check in the `return` as well? (currently also checking `listed`) https://github.com/zooniverse/pfe-lab/pull/135/files#diff-0c40371a8a91e7f3f917e73ef9dc299dR68

Reply:
I'm ok with `listed` on line 68 incase for whatever strange reason the org is changed to `listed: false` but the date doesn't get cleared out.